packages/app/routes/api/projects.js

```diff
@@ -66,68 +66,58 @@ router.get(
   async (req, res) => {
     log.info(
       {
-        requestId: req.id,
-        user: req.user.id
+        requestId: req.id
       },
       'User is getting projects'
     )
-    let projects = []
-    try {
-      projects = await Project.find()
-      if (req.query.name) {
-        projects = projects.filter(project =>
-          project.name.toLowerCase().includes(req.query.name.toLowerCase())
-        )
-      }
-      if (req.query.projectStage) {
-        projects = projects.filter(project =>
-          project.stageOfProject
-            .toLowerCase()
-            .includes(req.query.projectStage.toLowerCase())
-        )
-      }
-      if (req.query.innovationArea) {
-        projects = projects.filter(project =>
-          project.innovationCategory
-            .toLowerCase()
-            .includes(req.query.innovationArea.toLowerCase())
-        )
-      }
-      if (req.query.thematicArea) {
-        projects = projects.filter(project =>
-          project.thematicArea
-            .toLowerCase()
-            .includes(req.query.thematicArea.toLowerCase())
-        )
-      }
-      if (req.query.sort === 'asc') {
-        projects = projects.sort((a, b) =>
-          a.name > b.name ? 1 : b.name > a.name ? -1 : 0
-        )
-      } else {
-        projects = projects.sort((a, b) =>
-          a.name < b.name ? 1 : b.name < a.name ? -1 : 0
-        )
-      }
-      projects = projects.splice(req.query.offset, req.query.limit)
-      log.info(
-        {
-          requestId: req.id,
-          projects
-        },
-        'Success getting project list'
+    let projects = await Project.find()
+    if (req.query.name) {
+      projects = projects.filter(project =>
+        project.name.toLowerCase().includes(req.query.name.toLowerCase())
       )
-      return res.status(200).json({ projects })
-    } catch (error) {
-      log.info(
-        {
-          requestId: req.id,
-          error: error
-        },
-        'Can not get projects from the database'
+    }
+    if (req.query.projectStage) {
+      projects = projects.filter(project =>
+        project.stageOfProject
+          .toLowerCase()
+          .includes(req.query.projectStage.toLowerCase())
+      )
+    }
+    if (req.query.innovationArea) {
+      projects = projects.filter(project =>
+        project.innovationCategory
+          .toLowerCase()
+          .includes(req.query.innovationArea.toLowerCase())
+      )
+    }
+    if (req.query.thematicArea) {
+      projects = projects.filter(project =>
+        project.thematicArea
+          .toLowerCase()
+          .includes(req.query.thematicArea.toLowerCase())
       )
-      return sendError(res, 503, 'Error getting projects from the database')
```

Comment:
I think it has been removed when rebasing, @victor11555 did it before as a error handling, could you get back to this and put the "try catch" and this error handling, please?